packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/collection-header.twig

```diff
@@ -0,0 +1,22 @@
+{% set dark_font_class = collection.dark_font ? "collection-header--dark-font" : ""  %}
+
+<div class="collection-header {{ dark_font_class }}" style="background-color: {{ collection.bg_color }}">
+  <div class="collection-header--inner">
+
+    <div class="collection-header__content">
+      {% if collection.title is not empty %}
+        <div class="collection-header__title">{{ collection.title }}</div>
+      {% endif %}
+
+      {% if collection.description is not empty %}
+        <div class="collection-header__description">{{ collection.description }}</div>
+      {% endif %}
+    </div>
+
+    {% if collection.image|raw is not empty %}
+      <div class="collection-header__image">
+        {{ collection.image|raw }}
```

Comment:
@beto-aveiga Is there a reason we need to use raw here? I don't think this is something we've needed to do with other images in Mayflower.

Reply:
It was a mistake. I just removed it. Thanks!